cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/coprocessor/ConsumerConfigCache.java

```diff
@@ -198,18 +197,18 @@ public void run() {
             break;
           }
         }
-        LOG.info("Config cache update for {} terminated.", Bytes.toString(configTableName));
-        instances.remove(configTableName, this);
+        LOG.info("Config cache update for {} terminated.", Bytes.toString(queueConfigTableName));
+        instances.remove(queueConfigTableName, this);
       }
     };
     refreshThread.setDaemon(true);
     refreshThread.start();
   }
 
-  public static ConsumerConfigCache getInstance(Configuration hConf, byte[] tableName) {
+  public static ConsumerConfigCache getInstance(Configuration hConf, byte[] tableName, String sysConfigTablePrefix) {
     ConsumerConfigCache cache = instances.get(tableName);
     if (cache == null) {
-      cache = new ConsumerConfigCache(hConf, tableName);
+      cache = new ConsumerConfigCache(hConf, tableName, sysConfigTablePrefix);
```

Comment:
That is because the prefix is dependent on tableName (but I pass it in as second parameter, because it is computed outside of ConsumerConfigCache in a version-specific manner).